cmake_modules/commonSetup.cmake

```diff
@@ -76,13 +76,14 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   option(USE_UNSIGNED_CHAR "Build system with default char type is unsigned" OFF)
 
   option(USE_MYSQL "Enable MySQL support" ON)
-  option(USE_CASSANDRA "Enable Cassandra support" ON)
+  option(USE_CASSANDRA "Enable Cassandra support" OFF)
```

Comment:
Very sorry, I had stated this in the commit message. DON'T merge this PR please.